mucommander-core/src/main/java/com/mucommander/ui/action/impl/AbstractViewerAction.java

```diff
@@ -51,10 +51,10 @@ public AbstractViewerAction(MainFrame mainFrame, Map<String,Object> properties)
         super(mainFrame, properties);
 
         // Enable this action only if the currently selected file is not a directory and can be read.
-        setSelectedFileFilter(new AndFileFilter(
-            new FileOperationFilter(FileOperation.READ_FILE),
-            new AttributeFileFilter(FileAttribute.DIRECTORY, true)
-        ));
+        // setSelectedFileFilter(new AndFileFilter(
+        //     new FileOperationFilter(FileOperation.READ_FILE),
+        //     new AttributeFileFilter(FileAttribute.DIRECTORY, true)
+        // ));

```

Comment:
I guess we want to change the filter to folders or files that are readable rather than disabling it (so that this action would still be disabled for non-folders that are non-readable), right?

Reply:
fixed.